server/swap/swap.go

```diff
@@ -134,6 +141,12 @@ type stepActor struct {
 	status *swapStatus
 }
 
+// String satisfies the Stringer interface for pretty printing.
+func (a stepActor) String() string {
+	return fmt.Sprintf("user: %v, swapAsset: %v, isMaker: %v, order: %v, status: %v",
```

Comment:
Since `a.status` expands even more with it's own Stringer, this gets pretty difficult to read in the logs where it's used IMO.  Maybe `... status: {%v}` to group the status elements?